2014/01/16 10:56:15 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "gii".' in C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(141): CWebApplication->runController('gii')
#1 C:\xampp\htdocs\tokoapp\protected\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 C:\xampp\htdocs\tokoapp\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/tokoapp/index.php?r=gii
---
2014/01/16 10:58:05 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "gii".' in C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(141): CWebApplication->runController('gii')
#1 C:\xampp\htdocs\tokoapp\protected\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 C:\xampp\htdocs\tokoapp\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/tokoapp/index.php?r=gii
---
2014/01/17 12:29:37 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "gii/default/login".' in C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/default/log...')
#1 C:\xampp\htdocs\tokoapp\protected\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 C:\xampp\htdocs\tokoapp\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/tokoapp/index.php?r=gii/default/login
---
2014/01/17 12:47:02 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "gii/model".' in C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/model')
#1 C:\xampp\htdocs\tokoapp\protected\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 C:\xampp\htdocs\tokoapp\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/tokoapp/index.php?r=gii/model
---
2014/01/20 15:30:42 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection.connectionString cannot be empty.' in C:\xampp\htdocs\tokoapp\protected\framework\db\CDbConnection.php:370
Stack trace:
#0 C:\xampp\htdocs\tokoapp\protected\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 C:\xampp\htdocs\tokoapp\protected\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\tokoapp\protected\framework\base\CModule.php(387): CDbConnection->init()
#3 C:\xampp\htdocs\tokoapp\protected\framework\base\CModule.php(103): CModule->getComponent('db')
#4 C:\xampp\htdocs\tokoapp\protected\framework\gii\generators\model\ModelCode.php(60): CModule->__get('db')
#5 C:\xampp\htdocs\tokoapp\protected\framework\web\CFormModel.php(39): ModelCode->init()
#6 C:\xampp\htdocs\tokoapp\protected\framework\gii\CCodeGenerator.php(153): CFormModel->__construct()
#7 C:\xampp\htdocs\tokoapp\protected\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 C:\xampp\htdocs\tokoapp\protected\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(282): CController->run('index')
#13 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 C:\xampp\htdocs\tokoapp\protected\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 C:\xampp\htdocs\tokoapp\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/tokoapp/index.php?r=gii/model/index
HTTP_REFERER=http://localhost/tokoapp/index.php?r=gii/default/index
---
2014/01/20 15:31:09 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection.connectionString cannot be empty.' in C:\xampp\htdocs\tokoapp\protected\framework\db\CDbConnection.php:370
Stack trace:
#0 C:\xampp\htdocs\tokoapp\protected\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 C:\xampp\htdocs\tokoapp\protected\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\tokoapp\protected\framework\base\CModule.php(387): CDbConnection->init()
#3 C:\xampp\htdocs\tokoapp\protected\framework\base\CModule.php(103): CModule->getComponent('db')
#4 C:\xampp\htdocs\tokoapp\protected\framework\gii\generators\model\ModelCode.php(60): CModule->__get('db')
#5 C:\xampp\htdocs\tokoapp\protected\framework\web\CFormModel.php(39): ModelCode->init()
#6 C:\xampp\htdocs\tokoapp\protected\framework\gii\CCodeGenerator.php(153): CFormModel->__construct()
#7 C:\xampp\htdocs\tokoapp\protected\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 C:\xampp\htdocs\tokoapp\protected\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(282): CController->run('index')
#13 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 C:\xampp\htdocs\tokoapp\protected\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 C:\xampp\htdocs\tokoapp\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/tokoapp/index.php?r=gii/model/index
HTTP_REFERER=http://localhost/tokoapp/index.php?r=gii/controller/index
---
2014/01/20 15:31:59 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection.connectionString cannot be empty.' in C:\xampp\htdocs\tokoapp\protected\framework\db\CDbConnection.php:370
Stack trace:
#0 C:\xampp\htdocs\tokoapp\protected\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 C:\xampp\htdocs\tokoapp\protected\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\tokoapp\protected\framework\base\CModule.php(387): CDbConnection->init()
#3 C:\xampp\htdocs\tokoapp\protected\framework\base\CModule.php(103): CModule->getComponent('db')
#4 C:\xampp\htdocs\tokoapp\protected\framework\gii\generators\model\ModelCode.php(60): CModule->__get('db')
#5 C:\xampp\htdocs\tokoapp\protected\framework\web\CFormModel.php(39): ModelCode->init()
#6 C:\xampp\htdocs\tokoapp\protected\framework\gii\CCodeGenerator.php(153): CFormModel->__construct()
#7 C:\xampp\htdocs\tokoapp\protected\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 C:\xampp\htdocs\tokoapp\protected\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(282): CController->run('index')
#13 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 C:\xampp\htdocs\tokoapp\protected\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 C:\xampp\htdocs\tokoapp\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/tokoapp/index.php?r=gii/model/index
HTTP_REFERER=http://localhost/tokoapp/index.php?r=gii/controller/index
---
2014/01/20 15:33:05 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database 'tokoapp'' in C:\xampp\htdocs\tokoapp\protected\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\xampp\htdocs\tokoapp\protected\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 C:\xampp\htdocs\tokoapp\protected\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\tokoapp\protected\framework\base\CModule.php(387): CDbConnection->init()
#3 C:\xampp\htdocs\tokoapp\protected\framework\base\CModule.php(103): CModule->getComponent('db')
#4 C:\xampp\htdocs\tokoapp\protected\framework\gii\generators\model\ModelCode.php(60): CModule->__get('db')
#5 C:\xampp\htdocs\tokoapp\protected\framework\web\CFormModel.php(39): ModelCode->init()
#6 C:\xampp\htdocs\tokoapp\protected\framework\gii\CCodeGenerator.php(153): CFormModel->__construct()
#7 C:\xampp\htdocs\tokoapp\protected\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 C:\xampp\htdocs\tokoapp\protected\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(282): CController->run('index')
#13 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 C:\xampp\htdocs\tokoapp\protected\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 C:\xampp\htdocs\tokoapp\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/tokoapp/index.php?r=gii/model/index
---
2014/01/20 15:34:09 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database 'tokoapp'' in C:\xampp\htdocs\tokoapp\protected\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\xampp\htdocs\tokoapp\protected\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 C:\xampp\htdocs\tokoapp\protected\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\tokoapp\protected\framework\base\CModule.php(387): CDbConnection->init()
#3 C:\xampp\htdocs\tokoapp\protected\framework\base\CModule.php(103): CModule->getComponent('db')
#4 C:\xampp\htdocs\tokoapp\protected\framework\gii\generators\crud\CrudCode.php(44): CModule->__get('db')
#5 C:\xampp\htdocs\tokoapp\protected\framework\web\CFormModel.php(39): CrudCode->init()
#6 C:\xampp\htdocs\tokoapp\protected\framework\gii\CCodeGenerator.php(153): CFormModel->__construct()
#7 C:\xampp\htdocs\tokoapp\protected\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 C:\xampp\htdocs\tokoapp\protected\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(282): CController->run('index')
#13 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/crud/index')
#14 C:\xampp\htdocs\tokoapp\protected\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 C:\xampp\htdocs\tokoapp\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/tokoapp/index.php?r=gii/crud/index
HTTP_REFERER=http://localhost/tokoapp/index.php?r=gii/controller/index
---
2014/01/23 15:24:50 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested view "Biodata" was not found.' in C:\xampp\htdocs\tokoapp\protected\framework\web\actions\CViewAction.php:112
Stack trace:
#0 C:\xampp\htdocs\tokoapp\protected\framework\web\actions\CViewAction.php(123): CViewAction->resolveView('Biodata')
#1 C:\xampp\htdocs\tokoapp\protected\framework\web\actions\CAction.php(76): CViewAction->run()
#2 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(308): CAction->runWithParams(Array)
#3 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(286): CController->runAction(Object(CViewAction))
#4 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(265): CController->runActionWithFilters(Object(CViewAction), Array)
#5 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(282): CController->run('page')
#6 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(141): CWebApplication->runController('site/page')
#7 C:\xampp\htdocs\tokoapp\protected\framework\base\CApplication.php(180): CWebApplication->processRequest()
#8 C:\xampp\htdocs\tokoapp\index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/tokoapp/index.php?r=site/page&view=Biodata
HTTP_REFERER=http://localhost/tokoapp/index.php?r=Barang
---
2014/01/26 08:33:40 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested view "Biodata" was not found.' in C:\xampp\htdocs\tokoapp\protected\framework\web\actions\CViewAction.php:112
Stack trace:
#0 C:\xampp\htdocs\tokoapp\protected\framework\web\actions\CViewAction.php(123): CViewAction->resolveView('Biodata')
#1 C:\xampp\htdocs\tokoapp\protected\framework\web\actions\CAction.php(76): CViewAction->run()
#2 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(308): CAction->runWithParams(Array)
#3 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(286): CController->runAction(Object(CViewAction))
#4 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(265): CController->runActionWithFilters(Object(CViewAction), Array)
#5 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(282): CController->run('page')
#6 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(141): CWebApplication->runController('site/page')
#7 C:\xampp\htdocs\tokoapp\protected\framework\base\CApplication.php(180): CWebApplication->processRequest()
#8 C:\xampp\htdocs\tokoapp\index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/tokoapp/index.php?r=site/page&view=Biodata
HTTP_REFERER=http://localhost/tokoapp/
---
2014/01/26 08:45:12 [error] [exception.CException] exception 'CException' with message 'Alias "bootstrap.component.Bootstrap" is invalid. Make sure it points to an existing directory or file.' in C:\xampp\htdocs\tokoapp\protected\framework\YiiBase.php:347
Stack trace:
#0 C:\xampp\htdocs\tokoapp\protected\framework\YiiBase.php(196): YiiBase::import('bootstrap.compo...', true)
#1 C:\xampp\htdocs\tokoapp\protected\framework\base\CModule.php(386): YiiBase::createComponent(Array)
#2 C:\xampp\htdocs\tokoapp\protected\framework\base\CModule.php(523): CModule->getComponent('bootstrap')
#3 C:\xampp\htdocs\tokoapp\protected\framework\base\CApplication.php(163): CModule->preloadComponents()
#4 C:\xampp\htdocs\tokoapp\protected\framework\YiiBase.php(125): CApplication->__construct('C:\xampp\htdocs...')
#5 C:\xampp\htdocs\tokoapp\protected\framework\YiiBase.php(98): YiiBase::createApplication('CWebApplication', 'C:\xampp\htdocs...')
#6 C:\xampp\htdocs\tokoapp\index.php(13): YiiBase::createWebApplication('C:\xampp\htdocs...')
#7 {main}
REQUEST_URI=/tokoapp/
---
2014/01/26 08:46:58 [error] [exception.CException] exception 'CException' with message 'Alias "bootstrap.component.Bootstrap" is invalid. Make sure it points to an existing directory or file.' in C:\xampp\htdocs\tokoapp\protected\framework\YiiBase.php:347
Stack trace:
#0 C:\xampp\htdocs\tokoapp\protected\framework\YiiBase.php(196): YiiBase::import('bootstrap.compo...', true)
#1 C:\xampp\htdocs\tokoapp\protected\framework\base\CModule.php(386): YiiBase::createComponent(Array)
#2 C:\xampp\htdocs\tokoapp\protected\framework\base\CModule.php(523): CModule->getComponent('bootstrap')
#3 C:\xampp\htdocs\tokoapp\protected\framework\base\CApplication.php(163): CModule->preloadComponents()
#4 C:\xampp\htdocs\tokoapp\protected\framework\YiiBase.php(125): CApplication->__construct('C:\xampp\htdocs...')
#5 C:\xampp\htdocs\tokoapp\protected\framework\YiiBase.php(98): YiiBase::createApplication('CWebApplication', 'C:\xampp\htdocs...')
#6 C:\xampp\htdocs\tokoapp\index.php(13): YiiBase::createWebApplication('C:\xampp\htdocs...')
#7 {main}
REQUEST_URI=/tokoapp/
---
2014/01/26 09:04:58 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested view "Biodata" was not found.' in C:\xampp\htdocs\tokoapp\protected\framework\web\actions\CViewAction.php:112
Stack trace:
#0 C:\xampp\htdocs\tokoapp\protected\framework\web\actions\CViewAction.php(123): CViewAction->resolveView('Biodata')
#1 C:\xampp\htdocs\tokoapp\protected\framework\web\actions\CAction.php(76): CViewAction->run()
#2 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(308): CAction->runWithParams(Array)
#3 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(286): CController->runAction(Object(CViewAction))
#4 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(265): CController->runActionWithFilters(Object(CViewAction), Array)
#5 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(282): CController->run('page')
#6 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(141): CWebApplication->runController('site/page')
#7 C:\xampp\htdocs\tokoapp\protected\framework\base\CApplication.php(180): CWebApplication->processRequest()
#8 C:\xampp\htdocs\tokoapp\index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/tokoapp/index.php?r=site/page&view=Biodata
HTTP_REFERER=http://localhost/tokoapp/
---
2014/01/26 11:22:55 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested view "Biodata" was not found.' in C:\xampp\htdocs\tokoapp\protected\framework\web\actions\CViewAction.php:112
Stack trace:
#0 C:\xampp\htdocs\tokoapp\protected\framework\web\actions\CViewAction.php(123): CViewAction->resolveView('Biodata')
#1 C:\xampp\htdocs\tokoapp\protected\framework\web\actions\CAction.php(76): CViewAction->run()
#2 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(308): CAction->runWithParams(Array)
#3 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(286): CController->runAction(Object(CViewAction))
#4 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(265): CController->runActionWithFilters(Object(CViewAction), Array)
#5 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(282): CController->run('page')
#6 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(141): CWebApplication->runController('site/page')
#7 C:\xampp\htdocs\tokoapp\protected\framework\base\CApplication.php(180): CWebApplication->processRequest()
#8 C:\xampp\htdocs\tokoapp\index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/tokoapp/index.php?r=site/page&view=Biodata
HTTP_REFERER=http://localhost/tokoapp/index.php?r=site/index
---
2014/01/26 12:33:05 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested view "Biodata" was not found.' in C:\xampp\htdocs\tokoapp\protected\framework\web\actions\CViewAction.php:112
Stack trace:
#0 C:\xampp\htdocs\tokoapp\protected\framework\web\actions\CViewAction.php(123): CViewAction->resolveView('Biodata')
#1 C:\xampp\htdocs\tokoapp\protected\framework\web\actions\CAction.php(76): CViewAction->run()
#2 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(308): CAction->runWithParams(Array)
#3 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(286): CController->runAction(Object(CViewAction))
#4 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(265): CController->runActionWithFilters(Object(CViewAction), Array)
#5 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(282): CController->run('page')
#6 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(141): CWebApplication->runController('site/page')
#7 C:\xampp\htdocs\tokoapp\protected\framework\base\CApplication.php(180): CWebApplication->processRequest()
#8 C:\xampp\htdocs\tokoapp\index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/tokoapp/index.php?r=site/page&view=Biodata
HTTP_REFERER=http://localhost/tokoapp/index.php?r=site/index
---
2014/01/26 12:33:22 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested view "Biodata" was not found.' in C:\xampp\htdocs\tokoapp\protected\framework\web\actions\CViewAction.php:112
Stack trace:
#0 C:\xampp\htdocs\tokoapp\protected\framework\web\actions\CViewAction.php(123): CViewAction->resolveView('Biodata')
#1 C:\xampp\htdocs\tokoapp\protected\framework\web\actions\CAction.php(76): CViewAction->run()
#2 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(308): CAction->runWithParams(Array)
#3 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(286): CController->runAction(Object(CViewAction))
#4 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(265): CController->runActionWithFilters(Object(CViewAction), Array)
#5 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(282): CController->run('page')
#6 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(141): CWebApplication->runController('site/page')
#7 C:\xampp\htdocs\tokoapp\protected\framework\base\CApplication.php(180): CWebApplication->processRequest()
#8 C:\xampp\htdocs\tokoapp\index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/tokoapp/index.php?r=site/page&view=Biodata
HTTP_REFERER=http://localhost/tokoapp/index.php?r=site/contact
---
2014/01/26 12:35:54 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested view "Biodata" was not found.' in C:\xampp\htdocs\tokoapp\protected\framework\web\actions\CViewAction.php:112
Stack trace:
#0 C:\xampp\htdocs\tokoapp\protected\framework\web\actions\CViewAction.php(123): CViewAction->resolveView('Biodata')
#1 C:\xampp\htdocs\tokoapp\protected\framework\web\actions\CAction.php(76): CViewAction->run()
#2 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(308): CAction->runWithParams(Array)
#3 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(286): CController->runAction(Object(CViewAction))
#4 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(265): CController->runActionWithFilters(Object(CViewAction), Array)
#5 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(282): CController->run('page')
#6 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(141): CWebApplication->runController('site/page')
#7 C:\xampp\htdocs\tokoapp\protected\framework\base\CApplication.php(180): CWebApplication->processRequest()
#8 C:\xampp\htdocs\tokoapp\index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/tokoapp/index.php?r=site/page&view=Biodata
HTTP_REFERER=http://localhost/tokoapp/index.php?r=site/page&view=Biodata
---
2014/01/26 12:35:57 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested view "Biodata" was not found.' in C:\xampp\htdocs\tokoapp\protected\framework\web\actions\CViewAction.php:112
Stack trace:
#0 C:\xampp\htdocs\tokoapp\protected\framework\web\actions\CViewAction.php(123): CViewAction->resolveView('Biodata')
#1 C:\xampp\htdocs\tokoapp\protected\framework\web\actions\CAction.php(76): CViewAction->run()
#2 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(308): CAction->runWithParams(Array)
#3 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(286): CController->runAction(Object(CViewAction))
#4 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(265): CController->runActionWithFilters(Object(CViewAction), Array)
#5 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(282): CController->run('page')
#6 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(141): CWebApplication->runController('site/page')
#7 C:\xampp\htdocs\tokoapp\protected\framework\base\CApplication.php(180): CWebApplication->processRequest()
#8 C:\xampp\htdocs\tokoapp\index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/tokoapp/index.php?r=site/page&view=Biodata
HTTP_REFERER=http://localhost/tokoapp/index.php?r=site/page&view=Biodata
---
2014/01/26 12:36:45 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested view "Biodata" was not found.' in C:\xampp\htdocs\tokoapp\protected\framework\web\actions\CViewAction.php:112
Stack trace:
#0 C:\xampp\htdocs\tokoapp\protected\framework\web\actions\CViewAction.php(123): CViewAction->resolveView('Biodata')
#1 C:\xampp\htdocs\tokoapp\protected\framework\web\actions\CAction.php(76): CViewAction->run()
#2 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(308): CAction->runWithParams(Array)
#3 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(286): CController->runAction(Object(CViewAction))
#4 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(265): CController->runActionWithFilters(Object(CViewAction), Array)
#5 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(282): CController->run('page')
#6 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(141): CWebApplication->runController('site/page')
#7 C:\xampp\htdocs\tokoapp\protected\framework\base\CApplication.php(180): CWebApplication->processRequest()
#8 C:\xampp\htdocs\tokoapp\index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/tokoapp/index.php?r=site/page&view=Biodata
HTTP_REFERER=http://localhost/tokoapp/index.php?r=site/index
---
2014/01/26 12:37:53 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested view "Biodata" was not found.' in C:\xampp\htdocs\tokoapp\protected\framework\web\actions\CViewAction.php:112
Stack trace:
#0 C:\xampp\htdocs\tokoapp\protected\framework\web\actions\CViewAction.php(123): CViewAction->resolveView('Biodata')
#1 C:\xampp\htdocs\tokoapp\protected\framework\web\actions\CAction.php(76): CViewAction->run()
#2 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(308): CAction->runWithParams(Array)
#3 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(286): CController->runAction(Object(CViewAction))
#4 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(265): CController->runActionWithFilters(Object(CViewAction), Array)
#5 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(282): CController->run('page')
#6 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(141): CWebApplication->runController('site/page')
#7 C:\xampp\htdocs\tokoapp\protected\framework\base\CApplication.php(180): CWebApplication->processRequest()
#8 C:\xampp\htdocs\tokoapp\index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/tokoapp/index.php?r=site/page&view=Biodata
HTTP_REFERER=http://localhost/tokoapp/index.php?r=site/index
---
2014/01/27 10:30:10 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested view "Biodata" was not found.' in C:\xampp\htdocs\tokoapp\protected\framework\web\actions\CViewAction.php:112
Stack trace:
#0 C:\xampp\htdocs\tokoapp\protected\framework\web\actions\CViewAction.php(123): CViewAction->resolveView('Biodata')
#1 C:\xampp\htdocs\tokoapp\protected\framework\web\actions\CAction.php(76): CViewAction->run()
#2 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(308): CAction->runWithParams(Array)
#3 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(286): CController->runAction(Object(CViewAction))
#4 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(265): CController->runActionWithFilters(Object(CViewAction), Array)
#5 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(282): CController->run('page')
#6 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(141): CWebApplication->runController('site/page')
#7 C:\xampp\htdocs\tokoapp\protected\framework\base\CApplication.php(180): CWebApplication->processRequest()
#8 C:\xampp\htdocs\tokoapp\index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/tokoapp/index.php?r=site/page&view=Biodata
HTTP_REFERER=http://localhost/tokoapp/index.php?r=barang
---
2014/01/27 10:30:29 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested view "Biodata" was not found.' in C:\xampp\htdocs\tokoapp\protected\framework\web\actions\CViewAction.php:112
Stack trace:
#0 C:\xampp\htdocs\tokoapp\protected\framework\web\actions\CViewAction.php(123): CViewAction->resolveView('Biodata')
#1 C:\xampp\htdocs\tokoapp\protected\framework\web\actions\CAction.php(76): CViewAction->run()
#2 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(308): CAction->runWithParams(Array)
#3 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(286): CController->runAction(Object(CViewAction))
#4 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(265): CController->runActionWithFilters(Object(CViewAction), Array)
#5 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(282): CController->run('page')
#6 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(141): CWebApplication->runController('site/page')
#7 C:\xampp\htdocs\tokoapp\protected\framework\base\CApplication.php(180): CWebApplication->processRequest()
#8 C:\xampp\htdocs\tokoapp\index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/tokoapp/index.php?r=site/page&view=Biodata
HTTP_REFERER=http://localhost/tokoapp/index.php?r=barang
---
2014/01/27 10:31:05 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested view "Biodata" was not found.' in C:\xampp\htdocs\tokoapp\protected\framework\web\actions\CViewAction.php:112
Stack trace:
#0 C:\xampp\htdocs\tokoapp\protected\framework\web\actions\CViewAction.php(123): CViewAction->resolveView('Biodata')
#1 C:\xampp\htdocs\tokoapp\protected\framework\web\actions\CAction.php(76): CViewAction->run()
#2 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(308): CAction->runWithParams(Array)
#3 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(286): CController->runAction(Object(CViewAction))
#4 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(265): CController->runActionWithFilters(Object(CViewAction), Array)
#5 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(282): CController->run('page')
#6 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(141): CWebApplication->runController('site/page')
#7 C:\xampp\htdocs\tokoapp\protected\framework\base\CApplication.php(180): CWebApplication->processRequest()
#8 C:\xampp\htdocs\tokoapp\index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/tokoapp/index.php?r=site/page&view=Biodata
HTTP_REFERER=http://localhost/tokoapp/index.php?r=barang
---
2014/01/27 10:38:05 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested view "Biodata" was not found.' in C:\xampp\htdocs\tokoapp\protected\framework\web\actions\CViewAction.php:112
Stack trace:
#0 C:\xampp\htdocs\tokoapp\protected\framework\web\actions\CViewAction.php(123): CViewAction->resolveView('Biodata')
#1 C:\xampp\htdocs\tokoapp\protected\framework\web\actions\CAction.php(76): CViewAction->run()
#2 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(308): CAction->runWithParams(Array)
#3 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(286): CController->runAction(Object(CViewAction))
#4 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(265): CController->runActionWithFilters(Object(CViewAction), Array)
#5 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(282): CController->run('page')
#6 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(141): CWebApplication->runController('site/page')
#7 C:\xampp\htdocs\tokoapp\protected\framework\base\CApplication.php(180): CWebApplication->processRequest()
#8 C:\xampp\htdocs\tokoapp\index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/tokoapp/index.php?r=site/page&view=Biodata
HTTP_REFERER=http://localhost/tokoapp/index.php?r=site/contact
---
2014/01/27 10:39:06 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested view "Biodata" was not found.' in C:\xampp\htdocs\tokoapp\protected\framework\web\actions\CViewAction.php:112
Stack trace:
#0 C:\xampp\htdocs\tokoapp\protected\framework\web\actions\CViewAction.php(123): CViewAction->resolveView('Biodata')
#1 C:\xampp\htdocs\tokoapp\protected\framework\web\actions\CAction.php(76): CViewAction->run()
#2 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(308): CAction->runWithParams(Array)
#3 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(286): CController->runAction(Object(CViewAction))
#4 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(265): CController->runActionWithFilters(Object(CViewAction), Array)
#5 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(282): CController->run('page')
#6 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(141): CWebApplication->runController('site/page')
#7 C:\xampp\htdocs\tokoapp\protected\framework\base\CApplication.php(180): CWebApplication->processRequest()
#8 C:\xampp\htdocs\tokoapp\index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/tokoapp/index.php?r=site/page&view=Biodata
---
2014/01/27 10:39:08 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested view "about" was not found.' in C:\xampp\htdocs\tokoapp\protected\framework\web\actions\CViewAction.php:112
Stack trace:
#0 C:\xampp\htdocs\tokoapp\protected\framework\web\actions\CViewAction.php(123): CViewAction->resolveView('about')
#1 C:\xampp\htdocs\tokoapp\protected\framework\web\actions\CAction.php(76): CViewAction->run()
#2 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(308): CAction->runWithParams(Array)
#3 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(286): CController->runAction(Object(CViewAction))
#4 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(265): CController->runActionWithFilters(Object(CViewAction), Array)
#5 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(282): CController->run('page')
#6 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(141): CWebApplication->runController('site/page')
#7 C:\xampp\htdocs\tokoapp\protected\framework\base\CApplication.php(180): CWebApplication->processRequest()
#8 C:\xampp\htdocs\tokoapp\index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/tokoapp/index.php?r=site/page&view=about
HTTP_REFERER=http://localhost/tokoapp/index.php?r=site/page&view=Biodata
---
2014/01/27 10:39:14 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested view "about" was not found.' in C:\xampp\htdocs\tokoapp\protected\framework\web\actions\CViewAction.php:112
Stack trace:
#0 C:\xampp\htdocs\tokoapp\protected\framework\web\actions\CViewAction.php(123): CViewAction->resolveView('about')
#1 C:\xampp\htdocs\tokoapp\protected\framework\web\actions\CAction.php(76): CViewAction->run()
#2 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(308): CAction->runWithParams(Array)
#3 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(286): CController->runAction(Object(CViewAction))
#4 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(265): CController->runActionWithFilters(Object(CViewAction), Array)
#5 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(282): CController->run('page')
#6 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(141): CWebApplication->runController('site/page')
#7 C:\xampp\htdocs\tokoapp\protected\framework\base\CApplication.php(180): CWebApplication->processRequest()
#8 C:\xampp\htdocs\tokoapp\index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/tokoapp/index.php?r=site/page&view=about
HTTP_REFERER=http://localhost/tokoapp/index.php?r=site/page&view=about
---
2014/01/27 10:39:14 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested view "about" was not found.' in C:\xampp\htdocs\tokoapp\protected\framework\web\actions\CViewAction.php:112
Stack trace:
#0 C:\xampp\htdocs\tokoapp\protected\framework\web\actions\CViewAction.php(123): CViewAction->resolveView('about')
#1 C:\xampp\htdocs\tokoapp\protected\framework\web\actions\CAction.php(76): CViewAction->run()
#2 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(308): CAction->runWithParams(Array)
#3 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(286): CController->runAction(Object(CViewAction))
#4 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(265): CController->runActionWithFilters(Object(CViewAction), Array)
#5 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(282): CController->run('page')
#6 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(141): CWebApplication->runController('site/page')
#7 C:\xampp\htdocs\tokoapp\protected\framework\base\CApplication.php(180): CWebApplication->processRequest()
#8 C:\xampp\htdocs\tokoapp\index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/tokoapp/index.php?r=site/page&view=about
HTTP_REFERER=http://localhost/tokoapp/index.php?r=site/page&view=about
---
2014/01/27 10:40:52 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested view "about" was not found.' in C:\xampp\htdocs\tokoapp\protected\framework\web\actions\CViewAction.php:112
Stack trace:
#0 C:\xampp\htdocs\tokoapp\protected\framework\web\actions\CViewAction.php(123): CViewAction->resolveView('about')
#1 C:\xampp\htdocs\tokoapp\protected\framework\web\actions\CAction.php(76): CViewAction->run()
#2 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(308): CAction->runWithParams(Array)
#3 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(286): CController->runAction(Object(CViewAction))
#4 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(265): CController->runActionWithFilters(Object(CViewAction), Array)
#5 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(282): CController->run('page')
#6 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(141): CWebApplication->runController('site/page')
#7 C:\xampp\htdocs\tokoapp\protected\framework\base\CApplication.php(180): CWebApplication->processRequest()
#8 C:\xampp\htdocs\tokoapp\index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/tokoapp/index.php?r=site/page&view=about
HTTP_REFERER=http://localhost/tokoapp/index.php?r=site/page&view=about
---
2014/01/27 10:41:40 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "about".' in C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(270): CController->missingAction('about')
#1 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(282): CController->run('about')
#2 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(141): CWebApplication->runController('site/about')
#3 C:\xampp\htdocs\tokoapp\protected\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 C:\xampp\htdocs\tokoapp\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/tokoapp/index.php?r=site/about
HTTP_REFERER=http://localhost/tokoapp/index.php?r=site/page&view=about
---
2014/01/27 11:08:16 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "about".' in C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(270): CController->missingAction('about')
#1 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(282): CController->run('about')
#2 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(141): CWebApplication->runController('site/about')
#3 C:\xampp\htdocs\tokoapp\protected\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 C:\xampp\htdocs\tokoapp\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/tokoapp/index.php?r=site/about&site=about
HTTP_REFERER=http://localhost/tokoapp/
---
2014/01/27 11:39:50 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "index1".' in C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(270): CController->missingAction('index1')
#1 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(282): CController->run('index1')
#2 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(141): CWebApplication->runController('site/index1')
#3 C:\xampp\htdocs\tokoapp\protected\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 C:\xampp\htdocs\tokoapp\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/tokoapp/index.php?r=site/index1
---
2014/01/27 13:17:08 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "about".' in C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(270): CController->missingAction('about')
#1 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(282): CController->run('about')
#2 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(141): CWebApplication->runController('site/about')
#3 C:\xampp\htdocs\tokoapp\protected\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 C:\xampp\htdocs\tokoapp\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/tokoapp/index.php?r=site/about&site=about
HTTP_REFERER=http://localhost/tokoapp/index.php?r=barang/create
---
2014/01/27 13:20:30 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "about".' in C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(270): CController->missingAction('about')
#1 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(282): CController->run('about')
#2 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(141): CWebApplication->runController('site/about')
#3 C:\xampp\htdocs\tokoapp\protected\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 C:\xampp\htdocs\tokoapp\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/tokoapp/index.php?r=site/about&site=about
HTTP_REFERER=http://localhost/tokoapp/index.php?r=site/index
---
2014/01/27 13:31:44 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "about".' in C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(270): CController->missingAction('about')
#1 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(282): CController->run('about')
#2 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(141): CWebApplication->runController('site/about')
#3 C:\xampp\htdocs\tokoapp\protected\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 C:\xampp\htdocs\tokoapp\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/tokoapp/index.php?r=site/about&site=about
HTTP_REFERER=http://localhost/tokoapp/index.php?r=site/index
---
2014/01/27 13:32:22 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "about".' in C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(270): CController->missingAction('about')
#1 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(282): CController->run('about')
#2 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(141): CWebApplication->runController('site/about')
#3 C:\xampp\htdocs\tokoapp\protected\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 C:\xampp\htdocs\tokoapp\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/tokoapp/index.php?r=site/about&site=about
HTTP_REFERER=http://localhost/tokoapp/index.php?r=site/index
---
2014/01/27 13:39:40 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "about".' in C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\tokoapp\protected\framework\web\CController.php(270): CController->missingAction('about')
#1 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(282): CController->run('about')
#2 C:\xampp\htdocs\tokoapp\protected\framework\web\CWebApplication.php(141): CWebApplication->runController('site/about')
#3 C:\xampp\htdocs\tokoapp\protected\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 C:\xampp\htdocs\tokoapp\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/tokoapp/index.php?r=site/about&site=About
HTTP_REFERER=http://localhost/tokoapp/index.php?r=site/index
---
